k2/python/csrc/torch/torch_util.h

```diff
@@ -47,6 +47,7 @@ struct ToScalarType;
 
 // TODO(fangjun): add other types if needed
 TO_SCALAR_TYPE(float, torch::kFloat);
```

Comment:
The comments around ToScalarType do not really explain what it is or what it's needed for.

Reply:
It is used to convert a c type to an instance of `torch::ScalarType`. For example

```cpp
static_assert(torch::kFloat == ToScalarType<float>::value>, "");
static_assert(torch::kInt == ToScalarType<int32_t>::value>, "");

```

Reply:
Its usages are like the following:

https://github.com/k2-fsa/k2/blob/4efa9d21acb3798e33bb9b53efa743cff9b8f3f7/k2/python/csrc/torch/torch_util.cu#L77

Reply:
I think its meaning is clear from its definition:
https://github.com/k2-fsa/k2/blob/b118afd2be4dc8eded911d25c90e858dec63e8b7/k2/python/csrc/torch/torch_util.h#L43-L46